test/003_staking_rewards_test.js

```diff
@@ -126,6 +126,65 @@ describe("Staking Rewards Contract", function () {
         let updatedRewardPerToken = ethers.BigNumber.from(BigNumber.from(rewardRate).mul(timestamp - lastUpdateTime).mul(multiplier).div(totalSupply))
         expect(await stakingRewards.rewardPerToken()).to.be.eq(updatedRewardPerToken)
     });
+
+    it('Can update max reward', async function () {
+        rewardRateBeforeUpdateMaxReward = rewardRate;
+        await increaseTime(4 * DAY)
+        await expect(stakingRewards.connect(accounts[2]).updateMaxReward(BigNumber.from(10).pow(21))).to.be.reverted;
```

Comment:
just to confirm, this fails because we are calling function from a non-owner account?

Reply:
yes